AddInstallationScriptTest.cmake

```diff
@@ -0,0 +1,25 @@
+macro(add_installation_script_test name path)
```

Comment:
What if we run `get_filename_component(local_name ${name} NAME_WE)` so as to always strip off any training `.sh` if it is passed in to the macro, and then explicitly add it back in the macro? This has a pro and a con

**pro**: You can pass in the script name with or without the file extension/suffix

**con**: If you have scripts that don't end in `.sh` then when you write out the files below you will be removing whatever suffix was there, if any, and adding `.sh`.

You could get rid of the con by also calling `get_filename_componenet(orig_suffix ${name} EXT)` and then adding that back: `${local_name}${orig_suffix}`

Perhaps it might make sense to also sanitize the path with `get_filename_component(path_var ${path} DIRECTORY)` but I think we'd really have to be asleep at the wheel to pass in a file there... so that last suggestion is probably overkill